list.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] I would recommend implementing `#each` method that yields all members of @list in succession. In the project requirements, it is also mentioned. Below is the attached screenshot.

## Project requirement screenshot

![Screenshot from 2022-01-21 21-28-21](https://user-images.githubusercontent.com/22434660/150559164-1b8fd675-ac0f-4e38-ad62-d6485b58ea2d.png)